driver/compile.go

```diff
@@ -92,12 +87,15 @@ func compile(ctx context.Context, program ast.Proc, zctx *resolver.Context, read
 
 type MultiConfig struct {
 	Custom      compiler.Hook
+	Distributed bool // true if remote request specified worker count
```

Comment:
Do we need this, or could we just check for `Recruiter != "" || Workers != nil`?

Reply:
Let's address this later.